### PR TITLE
DOP-1860: Remove .live banner

### PR DESCRIPTION
**Pull request by rayangler**
[[Jira Ticket](https://jira.mongodb.org/browse/DOP-1860)]
[[Atlas Staging](https://jira.mongodb.org/browse/DOP-1860)]

I decided to keep the .live images in case we wanted to easily reuse it in the future, but I'm totally down to just delete it if it's not necessary to keep.

### CODE REVIEW

File: src/theme/docsTheme.js [hunk 1]
```diff
@@ -1,10 +1,5 @@
 // Set bannerContent = null when banner should be removed
```
Comment (sophstad) [inline src/theme/docsTheme.js:1]: I should've done this before, but how about a comment here specifying the object structure required to add a banner? We'll make lives easier for our future selves 🙌

**Review by sophstad — APPROVED** [review]
Wahoo! 💯

File: src/theme/docsTheme.js (hunk 1 above)
Comment (rayangler) [reply src/theme/docsTheme.js:1]: What do you think about the idea of adding an additional bool field `enabled` to `bannerContent`? In places where we check if `bannerContent` is null or not null, we check if `bannerContent.enabled` is `true`. This way, we can keep the banner details and fields up, and just change the bool field when we don't want it anymore.

**Review by sophstad — APPROVED** [review]
🎉

**Pull request merged**